### ISSUE TITLE
Maven 3.3.1 requires Java 7

### ISSUE THREAD

**Issue by BanzaiMan** [travis-ci/travis-ci#3778]
As such, the build will fail if run on Java 6.

**Comment by marcosnils** [comment]
@BanzaiMan indeed, I just came up with this issue also. It'd be great if you could either fallback to 3.2 for jdk1.6 or just allow to specify a maven version.

**Comment by BanzaiMan** [comment]
Thanks for the reports and we are really sorry for the issues. We are going to roll back the latest 2014-04 updates at 2015-04-28 1300 UTC. Cancel the 2014-04 updates altogether and concentrate on the next update and a better long-term planning.

See the details: http://stspg.io/10Su

**Comment by marcosnils** [comment]
@BanzaiMan thks for the update!

**Comment by manolama** [comment]
@BanzaiMan Thanks a bunch for the rollback!

**Comment by rmetzger** [comment]
Our builds were also affected by this issue.
Thank you for taking care of this.

**Comment by oldratlee** [comment]
+1 

CI for jdk 6 failed. 😢
